errors/ModuleNotFound.md

```diff
@@ -10,12 +10,24 @@ module ShortFailingExample where
 
 ## Cause
 
-Explain why a user might see this error.
+You might see this error because you have tried to import a Module that is not available on your system.
 
 ## Fix
 
 - Suggest possible solutions.
 
+Its likely you need to install the package which contains the module you are trying to use.
+
+A great place to go looking is https://pursuit.purescript.org where you can search for the Module by name, and find out which package you need to install.
+
+To install you'll want to use something like this:
+
+```
+bower install purescript-strings --save
+```
+
+You may need to use a different command, depending on how you have installed Purescript.
```

Comment:
The specific command doesn't really depend on how you have installed PureScript, but rather how your project is set up, I think.